terraform-deploy.sh

```
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Terraform Deployment Script
+# This script runs terraform plan and terraform apply for the SF Website infrastructure
+# Run this script from the project root directory
+
+set -e  # Exit on any error
+
+# Configuration from existing setup
+AWS_PROFILE="tf-sf-website"
+AWS_REGION="us-east-1"
```

Shellcheck (0.10.0)

[warning] AWS_REGION appears unused. Verify use (or export if used externally).

(SC2034)